Documentation - New plugin for frontend applications based in webpack (#5310)
hashicorp/consul: website/source/api/libraries-and-sdks.html.md

  <li>
    <a href="https://github.com/oatpp/oatpp-consul">oatpp-consul</a> - C++ Consul integration for <a href="https://oatpp.io/">oatpp</a> applications
  </li>
  <li>

>     <a href="https://github.com/marlonmleite/consul-env-webpack-plugin">consul-env-webpack-plugin</a> - Plugin for Webpack to convert KV-store (JSON) to environments variables in your front-end.

Reviewer: ```suggestion
    <a href="https://github.com/marlonmleite/consul-env-webpack-plugin">consul-env-webpack-plugin</a> - Webpack plugin for converting KV store JSON output to environments variables.
```
